api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -45,6 +45,12 @@ type NUMAResourcesOperatorSpec struct {
 	// +optional
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Optional ignore pod namespace/name glob patterns"
 	PodExcludes []NamespacedName `json:"podExcludes,omitempty"`
+	// Valid values are: "Normal", "Debug", "Trace", "TraceAll".
+	// Defaults to "Normal".
```

Comment:
nope. Defaults to whatever was given in the command line, which should be reported in status.